apps/dashboard/lib/rclone_util.rb

```diff
@@ -128,11 +128,20 @@ def touch(remote, path)
     end
 
     def mkdir(remote, path)
+      path = Pathname.new(path)
       full_path = remote_path(remote, path)
       o, e, s = rclone("mkdir", full_path)
       if e.include?("Warning: running mkdir on a remote which can't have empty directories does nothing")
-        # TODO: Could most likely do some kind of workaround here, e.g. rclone touch remote:path/.somefile
-        raise RcloneError.new(s.exitstatus), "Remote does not support empty directories"
+        # Workaround for remotes that don't support empty directories.
+        # Touch a .directory file in the directory that is being created
+        begin
+          touch(remote, path.join(".directory"))
```

Comment:
```suggestion
          touch(remote, path.join(".keep"))
```

`.keep` may only be a git convention, but I feel like it may be useful in the very likely case that someone is confused as to why this directory is not empty. They'll be able to at least google `.keep files` (with varying degrees of success).